### ISSUE TITLE
Default Creds

### ISSUE THREAD

**Issue by peasead** [yeti-platform/yeti#38]
### Description
I am embarrassed to ask, but I cannot find the default creds. It's certainly possible I missed a config file, but can't seem to find it.

I looked in `yeti/core/auth/local/user_management.py` and found:
```
def get_default_user():
    try:
        # Assume authentication is anonymous if only 1 user
        if User.objects.count() == 1:
            return User.objects.get(username="yeti")
        return AnonymousUserMixin()
    except DoesNotExist:
        return create_user("yeti", "yeti")
```
...but `yeti:yeti` didn't seem to work.

### Environment

| Question         | Answer
|------------------|--------------------
| Git commit       | 3a3dfa5983e8f1e245239eeb181af6a8f6742531
| OS version       | CentOS Linux release 7.3.1611 (Core)
| Browser          | Chrome 57.0.2987.110 (64-bit)

### Steps to Reproduce
Browse to `http://server:5000`
I attempted `admin:admin`, `admin:password`, `yeti:yeti`, and every combination in between.

#### Expected behavior
Log into the web app

#### Actual behavior
Invalid credentials

**Comment by splectrum** [comment]
I'm running into this problem as well, also on CentOS 7.  
I looked into the mongo database and saw a user collection, but it appears to be empty, which suggests nobody can login.


(checking users in mongo, user collection is empty)
> use yeti
switched to db yeti
> show collections
entity
export_template
indicator
link
malware_family
observable
schedule_entry
tag
user
> db.user.findOne();
null
>

**Comment by bryannolen** [comment]
I had the same problem. This little snippet of python will create a default user for you, allowing you to login and reset the password properly.

```
import core.auth.local.user_management
from core.user import User
from werkzeug.security import check_password_hash, generate_password_hash
u=User(username="yeti", permissions=core.auth.local.user_management.DEFAULT_PERMISSIONS)
u.password=generate_password_hash("yeti", method='pbkdf2:sha256:20000')
u.api_key = User.generate_api_key()
u.save()
```

**Comment by tomchop** [comment]
Sorry guys, this is me not testing my changes properly. Should be fixed in 27ae09f83b0165c36c78297708b4a2e75c9a2dad.
